engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreSubmitActor.scala

```diff
@@ -37,7 +37,9 @@ final case class WorkflowStoreSubmitActor(store: WorkflowStore, serviceRegistryA
 
       futureId onComplete {
         case Success(id) =>
-          log.info("Workflow {} submitted.", id)
+          val wfType = cmd.source.workflowType.getOrElse("Unspecified type")
+          val wfTypeVersion = cmd.source.workflowTypeVersion.getOrElse("Unspecified version")
+          log.info("{} ({}) workflow {} submitted", wfType, wfTypeVersion, id)
```

Comment:
ToL why not interpolate?

Reply:
this is more efficient as it doesn't interpolate if this is not logged